src/main.js

```diff
@@ -1,3 +1,16 @@
+// [X] On the new poster form view, users should be able to fill out the three input fields and then hit the save button
```

Comment:
We'll remove these and replace them with new goals when starting the next iteration.